index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
+
+  <body>
+<section id ="welcome-section">
+  <h1> Texto </h1>
+</section>
+<section id ="projects">
+  <h2 class="project-tile"> Projects</h2>
+  <a href="www.google.com"> link</a>
+  </section>
+  <nav id="navbar">
+    <a href="#welcome-section"> link dentro de la pagina</a>
+  </nav>
+  <a id="profile-link" href="https://github.com/montsecruz9" target="_blank"> Link a mi perfil </a>
+  </body>
+  
+

```

Comment:
🙌Please remember to remove the blank spaces, this will help to maintain good practices and will help the readability of the code.😊

```suggestion
 
```